src/digitaltwin/get_data_from_db.py

```diff
@@ -16,24 +16,28 @@ def get_data_from_db(engine, geometry: gpd.GeoDataFrame, source_list: tuple):
     get_data_from_apis.get_data_from_apis(engine, geometry, source_list)
     user_geometry = geometry.iloc[0, 0]
     poly = "'{}'".format(user_geometry)
-    for source in source_list: #when testing source_list = ['_50327-nz-river-centrelines']
-        # 2193 is an NZTM projection
+    for source in source_list:
+        #  when testing source_list = ['_50327-nz-river-centrelines']
+        #  2193 is an NZTM projection
```

Comment:
```suggestion```
Is this suggestion what you mean @caseyli92?